.gitignore

```diff
@@ -11,4 +11,5 @@ __pycache__
 /logs
 /registrar.json
 /dist
-chains
\ No newline at end of file
+chains
```

Comment:
This has my curiosity, and a quick bit of googling doesn't bed it back down.  What is `chains`?  Is this what I'm dealing with: http://populus.readthedocs.io/en/v1.3.0/chain.html

Reply:
Yes - That's the doc. Looks like this is ignoring a localchain config? Probably @szotov's